.github/workflows/deploy-artifact.yml

```diff
@@ -145,3 +145,18 @@ jobs:
           echo $IMAGE_ID
           docker tag $IMAGE_NAME $IMAGE_ID:$TAG
           docker push $IMAGE_ID:$TAG
+
+      - name: Update image and tag in Git
+        run : |
+          git config --global user.name "Release Bot"
+          git config --global user.email actions@github.com
+          git pull
+          cd extension-testcontainer/src/main/resources
+          IMAGE_ID=$OWNER/$IMAGE_NAME
+          sed -i '/container.image.name=/ s/=.*/=$IMAGE_ID/' config.properties
+          sed -i '/container.image.tag=/ s/=.*/=$TAG/' config.properties
+          if [[ `git status --porcelain --untracked-files=no` ]]; then
+            git commit -am "Update image and tag ($IMAGE_ID:$TAG)"
```

Comment:
🔧 Let's use the [commit-message guidelines](https://github.com/camunda-cloud/zeebe-process-test/blob/main/CONTRIBUTING.md#commit-message-guidelines)